asynchronous-authorization/langchain-next-js/package.json

```diff
@@ -22,68 +22,68 @@
-    "@radix-ui/react-popover": "^1.1.11",
-    "@radix-ui/react-slot": "^1.2.0",
-    "@types/pg": "^8.15.4",
+    "@auth0/ai-langchain": "^3.5.0",
+    "@auth0/nextjs-auth0": "^4.10.0",
+    "@langchain/community": "^0.3.55",
+    "@langchain/core": "^0.3.75",
+    "@langchain/langgraph": "^0.4.9",
+    "@langchain/openai": "^0.6.11",
+    "@radix-ui/react-avatar": "^1.1.10",
+    "@radix-ui/react-checkbox": "^1.3.3",
+    "@radix-ui/react-dialog": "^1.1.15",
+    "@radix-ui/react-dropdown-menu": "^2.1.16",
+    "@radix-ui/react-popover": "^1.1.15",
+    "@radix-ui/react-slot": "^1.2.3",
+    "@types/pg": "^8.15.5",
     "class-variance-authority": "^0.7.1",
     "clsx": "^2.1.1",
     "date-fns": "^4.1.0",
-    "dotenv": "^17.0.1",
-    "drizzle-orm": "^0.43.1",
-    "drizzle-zod": "^0.7.1",
-    "googleapis": "^148.0.0",
-    "langchain": "0.3",
-    "langgraph-nextjs-api-passthrough": "0.1",
-    "lucide-react": "^0.475.0",
-    "marked": "^15.0.7",
+    "drizzle-orm": "^0.44.5",
+    "drizzle-zod": "0.7.1",
+    "googleapis": "^159.0.0",
+    "langchain": "^0.3.33",
+    "langgraph-nextjs-api-passthrough": "^0.1.4",
+    "lucide-react": "^0.543.0",
+    "marked": "^16.2.1",
     "nanoid": "^5.1.5",
-    "next": "15.2.4",
-    "next-themes": "^0.4.4",
-    "nuqs": "^2.4.3",
+    "next": "^15.5.2",
+    "next-themes": "^0.4.6",
+    "nuqs": "^2.6.0",
     "pdf-parse": "^1.1.1",
     "pg": "^8.16.3",
-    "postgres": "^3.4.5",
-    "react": "19.0.0",
+    "postgres": "^3.4.7",
+    "react": "^19.1.1",
     "react-device-detect": "^2.2.3",
-    "react-dom": "19.0.0",
-    "react-markdown": "^10.0.0",
-    "react-toastify": "11.0.3",
-    "sonner": "^1.7.2",
-    "tailwind-merge": "^2.6.0",
+    "react-dom": "^19.1.1",
+    "react-markdown": "^10.1.0",
+    "react-toastify": "^11.0.5",
+    "sonner": "^2.0.7",
+    "tailwind-merge": "^3.3.1",
     "tailwindcss-animate": "^1.0.7",
-    "use-stick-to-bottom": "^1.0.44",
+    "use-stick-to-bottom": "^1.1.1",
     "vaul": "^1.1.2",
-    "zod": "^3.24.2",
+    "zod": "3.25.76 || ^4",
     "zod-to-json-schema": "^3.23.2"
   },
   "devDependencies": {
-    "@next/bundle-analyzer": "^15.1.7",
-    "@types/node": "^22.13.4",
+    "@next/bundle-analyzer": "^15.5.2",
+    "@types/node": "^24.3.1",
     "@types/pdf-parse": "^1.1.5",
-    "@types/react": "19.0.9",
-    "@types/react-dom": "19.0.3",
-    "autoprefixer": "^10.4.20",
-    "drizzle-kit": "^0.31.1",
-    "eslint": "^9.20.1",
-    "eslint-config-next": "^15.1.7",
+    "@types/react": "^19.1.12",
+    "@types/react-dom": "^19.1.9",
+    "autoprefixer": "^10.4.21",
+    "dotenv": "^16.4.5",
+    "drizzle-kit": "^0.31.4",
+    "eslint": "^9.35.0",
+    "eslint-config-next": "^15.5.2",
     "npm-run-all": "^4.1.5",
-    "postcss": "8.5.2",
-    "prettier": "^3.4.2",
-    "tailwindcss": "3.4.17",
-    "tsx": "^4.19.4",
-    "typescript": "5.7.3"
+    "postcss": "^8.5.6",
+    "prettier": "^3.6.2",
+    "@tailwindcss/postcss": "^4.1.13",
```

Comment:
tailwind 4 update seems to break rendering and compilation here. I suggest holding for future.